.github/workflows/ci-unit-test.yml

```diff
@@ -22,3 +22,58 @@ jobs:
 
     - name: Unit Tests
       run: python3 -m pytest
+
+    - name: Test execution from current working directory
+      run: python3 -m mkosi -h
+
+    - name: Test execution from current working directory (sudo call)
+      run: sudo python3 -m mkosi -h
+
+    - name: Test system installation
+      run: |
+        sudo python3 -m pip install .
```

Comment:
Does using `python3 -m pip` make a difference? I'd prefer testing with `pip3` since that's what I think most people will be using.

Reply:
Generally the standard for advising users goes in the direction of this version. A lot of Python people tend to have multiple Python installations and then it becomes unclear which pip relates to which python binary. Functionally doing `python3 -m pip` is exactly the same as directly calling `pip3` for the appropriate `python3` binary.